src/plugins/data/server/index_patterns/fetcher/index_patterns_fetcher.ts

```diff
@@ -58,16 +61,19 @@ export class IndexPatternsFetcher {
     rollupIndex?: string;
   }): Promise<FieldDescriptor[]> {
     const { pattern, metaFields, fieldCapsOptions, type, rollupIndex } = options;
+    const patternList = Array.isArray(pattern) ? pattern : pattern.split(',');
+    const patternListActive = await this.validatePatternListActive(patternList);
```

Comment:
Can we skip validating if there's only one item in the list?

Reply:
good idea thanks